clang/include/clang/Driver/Action.h

```diff
@@ -639,6 +640,17 @@ class SPIRVTranslatorJobAction : public JobAction {
   }
 };
 
+class SPIRCheckJobAction : public JobAction {
```

Comment:
please add a comment what this job is for

Reply:
Currently it is used only for FPGA-specific checking, but I think this action/tool is the place to add more checks of a SPIRV code in future if needed. - could be added to the comment too.